sensing/autoware_pointcloud_preprocessor/schema/ring_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Ring Outlier Filter Node",
+  "type": "object",
+  "definitions": {
+    "ring_outlier_filter": {
+      "type": "object",
+      "properties": {
+        "distance_ratio": {
+          "type": "number",
+          "description": "distance_ratio",
+          "default": "1.03"
```

Comment:
ranges

Reply:
applicable to several other parameters

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8468/commits/efb43daad85ee36ef70280c75d5517b07e306735